tests/test_metrics.py

```diff
@@ -21,177 +21,18 @@
-
-    @classmethod
-    def makeTask(cls, magMin=20, magMax=25):
-        """Make the task and allow for modification of the config min and max.
-
-        Parameters
-        ----------
-        magMin : min magnitude, `float`
-            Minimum magnitude
-        magMax : min magnitude, `float`
-            Maximum magnitude
-        """
-        config = ApFakesCountMetricConfig()
-        config.magMin = magMin
-        config.magMax = magMax
-
-        return ApFakesCountMetricTask(config=config)
-
-    def setUp(self):
-        super().setUp()
-
-        simpleMapConfig = skyMap.discreteSkyMap.DiscreteSkyMapConfig()
-        simpleMapConfig.raList = [45]
-        simpleMapConfig.decList = [45]
-        simpleMapConfig.radiusList = [0.1]
-
-        self.simpleMap = skyMap.DiscreteSkyMap(simpleMapConfig)
-        self.tractId = 0
-        bCircle = self.simpleMap.generateTract(self.tractId).getOuterSkyPolygon().getBoundingCircle()
-        self.targetSources = 1000
-        self.sourceDensity = (self.targetSources
-                              / (bCircle.getArea() * (180 / np.pi) ** 2))
-
-        fakesConfig = CreateRandomApFakesConfig()
-        fakesConfig.fraction = 0.0
-        fakesConfig.fakeDensity = self.sourceDensity
-        fakesTask = CreateRandomApFakesTask(config=fakesConfig)
-        fakeCat = fakesTask.run(self.tractId, self.simpleMap).fakeCat
-
-        self.band = 'g'
-        self.magCut = 25
-        magMask = (fakeCat[fakesConfig.mag_col % self.band] < self.magCut)
-        self.expectedAllMatched = magMask.sum()
-        ids = np.where(magMask, np.arange(1, len(fakeCat) + 1, dtype=int), 0)
-        # Add columns to mimic the matched fakes result without running the
-        # full pipeline.
-        self.fakeCat = fakeCat.assign(diaObjectId=ids,
-                                      filterName=["g"] * len(fakeCat),
-                                      diaSourceId=ids)
-
-    def testValid(self):
-        """Test the run method.
-        """
-        result = self.task.run(self.fakeCat, self.band)
-        testUtils.assertValidOutput(self.task, result)
-
-        meas = result.measurement
-        self.assertEqual(meas.metric_name, Name(metric="ap_pipe.apFakesCompleteness"))
-        # Work around for Mac failing this test.
-        self.assertAlmostEqual(
-            meas.quantity.value,
-            (self.expectedAllMatched * u.count).value,
-            places=2)
-
-    def testValidEmpty(self):
-        """Test the run method with a valid but zero result.
-        """
-        # Make a mag cut that will have no sources. 30 < g < 35.
-        metricComplete = self.makeTask(self.magCut + 5, self.magCut + 10)
-        result = metricComplete.run(self.fakeCat, self.band)
-        testUtils.assertValidOutput(metricComplete, result)
-
-        meas = result.measurement
-        self.assertEqual(meas.metric_name, Name(metric="ap_pipe.apFakesCompleteness"))
-        self.assertEqual(meas.quantity, 0 * u.count)
```

Comment:
Please do this removal on the same commit as "Removed metrics that are...". Otherwise, the prior commit leaves the repo in an inconsistent state, which [can cause problems when examining history or moving/cherry-picking commits](https://developer.lsst.io/work/flow.html#appendix-commit-organization-best-practices). Naturally, this goes for adding tests as well.